code42_connector.py

```diff
@@ -491,11 +503,30 @@ def _handle_list_cases(self, param, action_result):
     def _handle_add_case_event(self, param, action_result):
         case_number = param["case_number"]
         event_id = param["event_id"]
-        self._client.cases.file_events.add(case_number=case_number, event_id=event_id)
+        try:
+            self._client.cases.file_events.add(
+                case_number=case_number, event_id=event_id
+            )
+        except Py42BadRequestError as err:
+            if "NO_SUCH_CASE" in err.response.text:
+                self._handle_case_not_found(err, case_number)
+            elif "NO_SUCH_EVENT" in err.response.text:
+                message = f"Event ID {event_id} not found."
+                err.args = (message,)
+                raise err
+            else:
+                raise err
```

Comment:
This should just be `raise` instead of `raise err` to preserve the exception stack